### PR TITLE
Task/csi 2691 printing the version in the logs

### PR DESCRIPTION
**Pull request by zingero**
Controller:
`2021-04-11 20:23:33,916 INFO    [139949670514944] [MainThread] (csi_controller_server.py:start_server:753) - Controller version: 1.6.0`

Node:
`2021-04-11 20:21:57,4118 INFO   [1] [-] (main.go:33) - Node version: {
  "driverVersion": "1.6.0",
  "gitCommit": "fc3e67732bd318097aa31fabacd6791c79f729ed",
  "buildDate": "2021-04-11T20:16:09Z",
  "goVersion": "go1.12.6",
  "compiler": "gc",
  "platform": "linux/amd64"
}`

### CODE REVIEW

File: node/cmd/main.go [hunk 1]
```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/ibm/ibm-block-csi-driver/node/pkg/driver"
 )
 
+func printVersion(configFile *string) {
```
Comment (oriyarde) [inline node/cmd/main.go:28]: looks more like `printVersionInfo` to me (same for log message)

File: node/cmd/main.go [hunk 2]
```diff
@@ -25,6 +25,14 @@ import (
...
+	info, err := driver.GetVersionJSON(*configFile)
+	if err != nil {
+		logger.Panicln(err)
```
Comment (oriyarde) [inline node/cmd/main.go:31]: since the program continues normally, I think "error" is enough

File: node/cmd/main.go [hunk 3]
```diff
@@ -36,12 +44,9 @@ func main() {
 
 	flag.Parse()
 
+	printVersion(configFile)
+
 	if *version {
```
Comment (oriyarde) [inline node/cmd/main.go:49]: `exitAfterLogVersion`?

File: node/cmd/main.go (hunk 1 above)
Comment (zingero) [reply node/cmd/main.go:28]: Done

File: node/cmd/main.go (hunk 2 above)
Comment (zingero) [reply node/cmd/main.go]: Done

File: node/cmd/main.go (hunk 3 above)
Comment (zingero) [reply node/cmd/main.go]: Done

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**